director/builder.py

```diff
@@ -11,82 +13,85 @@
 
 class WorkflowBuilder(object):
     def __init__(self, workflow_id):
+        self.workflow = Workflow.query.filter_by(id=workflow_id).first()
+        self.blueprint = cel_workflows.get_by_name(self.workflow)
```

Comment:
Still think "schema" might be a better name, and make more sense to developers...